embargoes.install

```diff
@@ -56,3 +61,18 @@ function embargoes_update_8001(&$sandbox) {
     'type' => 'int',
   ]);
 }
+
+/**
+ * Update the 'action' column to use integers. Add indices.
+ */
+function embargoes_update_8002(&$sandbox) {
+  $schema = Database::getConnection()->schema();
+  $schema->changeField('embargoes_log', 'action', 'action', [
+    'type' => 'int',
+    'size' => 'tiny',
+    'not null' => TRUE,
+  ]);
+  $schema->addIndex('embargoes_log', 'action', ['action']);
```

Comment:
`[error]  Too few arguments to function Drupal\Core\Database\Driver\pgsql\Schema::addIndex(), 3 passed in /opt/www/drupal/modules/contrib/embargoes/embargoes.install on line 75 and exactly 4 expected`